src/main/java/hudson/plugins/throttleconcurrents/ThrottleJobProperty.java

```diff
@@ -319,20 +322,34 @@ public ListBoxModel doFillCategoryItems() {
         private Integer maxConcurrentPerNode;
         private Integer maxConcurrentTotal;
         private String categoryName;
+        private String blockingCategories;
+        private List<String> blockingCategoriesList = new ArrayList<String>();
         private List<NodeLabeledPair> nodeLabeledPairs;
 
         @DataBoundConstructor
         public ThrottleCategory(String categoryName,
```

Comment:
Please save the previous constructor as well. Otherwise the change affects the binary compatibility with previous versions